logger/logger.md

```diff
@@ -1,20 +1,38 @@
 # GarageLogger
+Proposal by Peter Lillian
 
-The current logger is simply a collection of global variables that needs to be replaced with a new class. I'm calling this new class **GarageLogger**.
+## Background
 
-![GarageLogger](GarageLogger.svg)
+The current logger is simply a collection of global variables that needs to be replaced with a new object-oriented API that is easier to work with. I'm calling this new class **GarageLogger**.
 
-I'm planning on using [jruere/multiprocessing-logging](https://github.com/jruere/multiprocessing-logging) to handle multiprocessing. This will make it much easier for all your processes to log to the same place: essentially all it does is wrap a python logger so it plays well in a multiprocessing environment. GarageLogger will abstract all of this away so you don't have to worry about it.
+**Problem Statement**
+The logger interface is a bolt-on, package-gobal mess that isn't multiprocess-aware and fails to conform to garage code 
 
-The new logger will have one `add_output` method that takes in both the type of output and any arguments needed for that type. Each output will be its own implementation of an **OutputType** abstract class, which will contain a multiprocessing logger and handle writing to any outputs assigned. GarageLogger will keep a dictionary of the current OutputTypes. For example: **TextOutput**, **TabularOutput**, or **TensorboardOutput**
+**Scope**
+This PR encompasses creating a usable logger API for garage that is easily extensible and editable.
```

Comment:
I think this will take many PRs.

The scope should talk about what the logger will and won't do.